shared/src/commonTest/kotlin/co/touchlab/kampstarter/BreedModelTest.kt

```diff
@@ -25,39 +30,46 @@ class BreedModelTest: BaseTest() {
         appStart(dbHelper, settings, ktorApi, kermit)
         dbHelper.deleteAll()
 
-        model = BreedModel(viewUpdate = { summary ->
-            if (summary.allItems.isNotEmpty())
-                itemDataSummary.complete(summary)
-        }, errorUpdate = { s ->
+        model = BreedModel(errorUpdate = { s ->
             errorString.complete(s)
         })
+
+        itemDataSummary.complete(model.selectAllBreeds().first())
     }
 
     @Test
     fun staleDataCheckTest() = runTest {
         settings.putLong(BreedModel.DB_TIMESTAMP_KEY, currentTimeMillis())
         assertFalse(ktorApi.jsonRequested)
-        model.getBreedsFromNetwork().join()
+        val deferred = async { model.getBreedsFromNetwork() }
+        deferred.await()
         assertFalse(ktorApi.jsonRequested)
     }
 
     @Test
     fun updateFavoriteTest() = runTest {
-        model.getBreedsFromNetwork().join()
+
+        val deferred = async { model.getBreedsFromNetwork() }
+        deferred.await()
         itemDataSummary.await(500)
-        val breedOld = dbHelper.selectAllItems().executeAsList().first()
+        val breedOld = dbHelper.selectAllItems().first().first()
```

Comment:
These `first().first()` calls are a little funny-looking but doesn't really matter

Reply:
Yea it's the first in the list taken from the first in flow. Could separate the flow and list but would just be for visuals.